proto/aserto/directory/model/v3/model.proto

```diff
@@ -2,25 +2,118 @@ syntax = "proto3";
+                }
+            }
+        };
+    };
+	rpc ListManifests(ListManifestsRequest) returns (ListManifestsResponse) {
+        option (google.api.http) = {
+            get: "/api/v3/directory/manifests"
+        };
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            tags: "directory"
+            summary: "List manifests"
+            description: "Returns list of stored manifests."
+            operation_id: "directory.v3.manifests.list"
+            deprecated: false
+            security: {
+                security_requirement: {
+                    key: "TenantID";
+                    value: {}
+                }
+                security_requirement: {
+                    key: "DirectoryAPIKey";
+                    value: {}
+                }
+            }
+        };
+	};
+
+	rpc GetObjectType(GetObjectTypeRequest) returns (GetObjectTypeResponse) {
+        option (google.api.http) = {
+            get: "/api/v3/directory/object_type/{name}"
+        };
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            tags: "directory"
+            summary: "Get object type"
+            description: "Returns single object type."
+            operation_id: "directory.v3.object_types.list"
+            deprecated: false
+            security: {
+                security_requirement: {
+                    key: "TenantID";
+                    value: {}
+                }
+                security_requirement: {
+                    key: "DirectoryAPIKey";
+                    value: {}
+                }
+            }
+        };
+	};
+	rpc GetObjectTypes(GetObjectTypesRequest) returns (GetObjectTypesResponse) {
+        option (google.api.http) = {
+            get: "/api/v3/directory/object_types"
+        };
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            tags: "directory"
+            summary: "List all object types"
+            description: "Returns list of object types."
+            operation_id: "directory.v3.object_types.list"
+            deprecated: false
+            security: {
+                security_requirement: {
+                    key: "TenantID";
+                    value: {}
+                }
+                security_requirement: {
+                    key: "DirectoryAPIKey";
+                    value: {}
+                }
+            }
+        };
+	};
 }
 
 message GetManifestRequest {
-    // manifest name (unique, lc-string)
-    string name                                                 = 1     [(google.api.field_behavior) = REQUIRED];
-    // manifest version (semver notation)
-    string version                                              = 2     [(google.api.field_behavior) = REQUIRED];
+   // manifest name (unique, lc-string)
+   string  name                                                 = 1      [(google.api.field_behavior) = REQUIRED];
```

Comment:
Perhaps name and version should be optional to allow retrieving the latest manifest if they're not provided, and the latest version if only name is specified.
@gertd any thoughts on that?

Reply:
The version I can see being optional and returning the latest, in that case, name not as there is no default name, nor the assumption there is only one manifest at all times.